packages/components/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 -   `Radio`: Deprecate 36px default size ([#66572](https://github.com/WordPress/gutenberg/pull/66572)).
 
+### Bug Fixes
+
+-   `ToggleGroupControl`: Fix active background for `zero` value ([#66855](https://github.com/WordPress/gutenberg/pull/66855)).
```

Comment:
I feel like this would be clearer?

```suggestion
-   `ToggleGroupControl`: Fix active background for `0` value ([#66855](https://github.com/WordPress/gutenberg/pull/66855)).
```